registry/remote/example_test.go

```diff
@@ -45,9 +45,22 @@ const (
 )
 
 var (
-	exampleLayerDigest    = digest.FromBytes([]byte(exampleLayer)).String()
-	exampleManifestDigest = digest.FromBytes([]byte(exampleManifest)).String()
-	exampleRefereceDigest = "sha256:b2122d3fd728173dd6b68a0b73caa129302b78c78273ba43ead541a88169c855"
+	exampleLayerDigest        = digest.FromBytes([]byte(exampleLayer)).String()
+	exampleManifestDigest     = digest.FromBytes([]byte(exampleManifest)).String()
+	exampleManifestDescriptor = artifactspec.Descriptor{
+		MediaType: artifactspec.MediaTypeArtifactManifest,
```

Comment:
It's better to use `ocispec.MediaTypeImageManifest` here and be consistent with [the push example](https://github.com/oras-project/oras-go/pull/173/files).

Reply:
changed.